### PR TITLE
adding dodas documentation

### PR DESCRIPTION
**Pull request by spigad**
# Summary

---

**Related issue :**

### CODE REVIEW

**Comment by gwarf** [comment]
Thanks @spigad !
Please update the file front matter to remove the `draft: true` line, and complete the short description.

**Comment by spigad** [comment]
Done.

**Comment by gwarf** [comment]
@spigad once the linter will be happy we will enable the preview of your changes, thank!
See https://github.com/EGI-Federation/documentation/runs/5623927740?check_suite_focus=true#step:4:11826

**Comment by gwarf** [comment]
I've merged #426 that will fix the report from textlint, so this one should go if you update the branch.

**Comment by enolfc** [comment]
Thanks @spigad! Could you reformat the file to avoid the very long lines? You can use prettier as described in https://docs.egi.eu/about/contributing/style/#automating-formatting-and-checking

This is the command line I normally use for that
```
$ npx prettier --write --print-width 80 --tab-width 2 --prose-wrap always .
```

**Comment by spigad** [comment]
Enol, it should be ok now. let  me know

**Comment by gwarf** [comment]
The DODAS Preview is here: https://docs.egi.eu/documentation/425/users/compute/orchestration/dodas/

**Review by enolfc — CHANGES_REQUESTED** [review]
I think this is good as starting point.

Could you also modify https://github.com/EGI-Federation/documentation/blob/main/content/en/users/compute/orchestration/_index.md?plain=1#L31 and uncomment DODAS entry and fix the text of the description?

**Comment by enolfc** [comment]
@spigad can you check this one:
> Could you also modify https://github.com/EGI-Federation/documentation/blob/main/content/en/users/compute/orchestration/_index.md?plain=1#L31 and uncomment DODAS entry and fix the text of the description?

**Comment by thebe14** [comment]
I am looking at the first paragraph of "what is it?" and it is not clear to me (from reading that) what dodas is. could you separate the features of dodas and what it allows in subsequent paragraphs and keep the first paragraph short, concise, that says clearly what dodas is?

File: content/en/users/compute/orchestration/dodas/_index.md [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
 weight: 40
 description: >
-  Subtitle here
+  Instantiate on-demand customizable infrastructures for data analysis
 ---
 
 ## What is it?
 
-TBD
+DODAS enables the execution of user analysis code both in batch mode and
```
Comment (thebe14) [inline content/en/users/compute/orchestration/dodas/_index.md:12]: Do not use acronyms without introducing them first. On the first occurrence in the text*, use the full name (in this case "Dynamic On-Demand Analysis Software") followed by the acronym in between parenthesis. The first full use of the service name should also be a link to the actual service**.

* in titles always use the full name
** never put links in titles, though

File: content/en/users/compute/orchestration/dodas/_index.md [hunk 2]
```diff
@@ -1,13 +1,36 @@
...
+several building blocks that can be combined together in order to create the
+best service composition for a given use case. The currently available blocks
+allow to combine Jupyter and HTCondor as well as Jupyter and Spark or simply a
+jupyter interface. In addition, they allow the management of data via caches to
+optimize the processing of remote data. This can be done either via XCache or
+MinIO S3 object storage capabilities. DODAS is based on docker containers and
+the related orchestration relies on Kubernetes that enables the possibility to
+compose the building blocks via a web-based user interface thanks to Kubeapps.
+
+In order to deploy it services over various backend, DODAS relies on
```
Comment (thebe14) [inline content/en/users/compute/orchestration/dodas/_index.md:23]: The PaaS link is broken. Also, do not link to a draft page from a non-draft page.

File: content/en/users/compute/orchestration/dodas/_index.md [hunk 3]
```diff
@@ -1,13 +1,36 @@
...
+[Infrastructure Manager](../im) and [INDIGO PaaS](../indigo-paas)
+
+{{% alert title="Tip" color="info" %}} If you want to give it a try and deploy
```
Comment (thebe14) [inline content/en/users/compute/orchestration/dodas/_index.md:26]: Maybe this tip can go a use the link for the first occurrence of the service name.

**Comment by thebe14** [comment]
As it is now, this page is just like an ad on TV for a product. In my opinion, it needs to become the actual documentation on how to use the service (create subpages as needed). And only refer to its full blown documentation when advanced features are to be used. Basic usage should be possible by only looking at this page and its subpages.

File: content/en/users/compute/orchestration/dodas/_index.md (hunk 1 above)
Comment (enolfc) [inline content/en/users/compute/orchestration/dodas/_index.md:12]:
```suggestion
[Dynamic On-Demand Analysis Software (DODAS)](https://dodas-iam.cloud.cnaf.infn.it/login) 
enables the execution of user analysis code both in batch mode and
```

File: content/en/users/compute/orchestration/dodas/_index.md [hunk 4]
```diff
@@ -1,13 +1,36 @@
...
+best service composition for a given use case. The currently available blocks
+allow to combine Jupyter and HTCondor as well as Jupyter and Spark or simply a
+jupyter interface. In addition, they allow the management of data via caches to
+optimize the processing of remote data. This can be done either via XCache or
+MinIO S3 object storage capabilities. DODAS is based on docker containers and
+the related orchestration relies on Kubernetes that enables the possibility to
+compose the building blocks via a web-based user interface thanks to Kubeapps.
+
+In order to deploy it services over various backend, DODAS relies on
+[Infrastructure Manager](../im) and [INDIGO PaaS](../indigo-paas)
```
Comment (enolfc) [inline content/en/users/compute/orchestration/dodas/_index.md:24]:
```suggestion
[Infrastructure Manager](../im) and
[INDIGO PaaS](https://indigo-paas.cloud.ba.infn.it/home/login)
```

File: content/en/users/compute/orchestration/dodas/_index.md (hunk 3 above)
Comment (enolfc) [reply content/en/users/compute/orchestration/dodas/_index.md:26]: Keeping it until we have a better documentation

**Review by enolfc — APPROVED** [review]
:+1: Approving this one to have an initial documentation available. Should be improved in the future with more details on how DODAS plays with the EGI Cloud

File: content/en/users/compute/orchestration/dodas/_index.md [hunk 5]
```diff
@@ -1,13 +1,38 @@
...
 weight: 40
 description: >
-  Subtitle here
+  Instantiate on-demand customizable infrastructures for data analysis
 ---
 
 ## What is it?
 
-TBD
+[Dynamic On-Demand Analysis Software (DODAS)](https://dodas-iam.cloud.cnaf.infn.it/login)
```
Comment (enolfc) [inline content/en/users/compute/orchestration/dodas/_index.md:12]:
```suggestion
[Dynamic On-Demand Analysis Software (DODAS)](https://dodas-iam.cloud.cnaf.infn.it/login)
```

**Pull request merged**